cmd/apiserver/app/server/server.go

```diff
@@ -52,10 +52,10 @@ const (
 	// I made this up to match some existing paths. I am not sure if there
 	// are any restrictions on the format or structure beyond text
 	// separated by slashes.
-	etcdPathPrefix = "/k8s.io/incubator/service-catalog"
```

Comment:
Just for the record, we don't want to put 'incubator' in the name so that we don't have to migrate users later if this moves to the kubernetes org.